man/systemd.special.xml

```diff
@@ -104,942 +104,965 @@
   </refsect1>
```

Comment:
Hmm, you say: 

> I'm using "System manager units" and "user manager units" instead of the more
obvious "system units" and "user units", because there are also units like
"user@.service"."

Well, but if you ask me what `user@.service` is then I'd reply it's the "unit containing the user manager", or shorter "user manager unit". Which hence creates a lot of confusion.

Maybe be explicit:

"Units managed by the system's service manager"
"Units managed by the user's service manager"

That way I think it's clear who manages what